.gitlab/functional_test/security_agent.yml

```diff
@@ -30,8 +44,10 @@ kitchen_test_security_agent_x64:
   variables:
     KITCHEN_ARCH: x86_64
   before_script:
-    - cd $DD_AGENT_TESTING_DIR
+    - pushd $DD_AGENT_TESTING_DIR
     - tasks/kitchen_setup.sh
+  script:
+    - !reference [ .runtime, script ]
```

Comment:
Nit: is this needed?

This job extends `.kitchen_test_security_agent`, and `.kitchen_test_security_agent` already has:
```
  script:
    - !reference [.runtime, script]
 ```

Reply:
good catch!